### DISCUSSION TITLE
Custom system prompt for calls

### DISCUSSION THREAD

**Discussion by ahmedsaed** [open-webui/open-webui#5859] [Ideas]
**Is your feature request related to a problem? Please describe.**

The LLM occasionally generates responses that are inappropriate for call interactions. For instance:

> User: Hello!

> LLM: It's so great to finally meet you in person (virtually, of course). I hope you're having an amazing day so far. If not, don't worry - we can turn that around with some knowledge, laughter, and maybe a few bad jokes.
> 
> What's on your mind? Got a question, need help, or just want to chat about something completely random? I'm all ears (or in this case, all text). Fire away!

In this example, the LLM adds unnecessary clarifications and mentions text interactions, which are not suitable for calls.

**Describe the solution you'd like**

A custom system prompt tailored specifically for calls would be more effective. This prompt could be dynamically appended to the main system prompt, allowing the LLM to adjust its tone and responses for calls without redundant information.

**Describe alternatives you've considered**

Another option is to create multiple models within the workspace, each with a custom system prompt suited to different scenarios.

**Comment by spammenotinoz** [comment]
Good concept, I attempted to look at using a specific model for the "Call" function, was going to do this via Functions, but really couldn't work out how to Identify if a user invoked "Call".

In the interim I have a specific voice model, where I appended "Please provide concise responses to questions or prompts. Aim for clarity and brevity, keeping answers under 40 words" to the system prompt.

**Comment by lee-b** [comment]
There is a setting for this: Admin Panel -> Settings -> Interface -> Voice Mode Custom Prompt toggle, and the setting below that for the content of the prompt (if you don't want to use the default).

HOWEVER, can anyone confirm that this doesn't work for them either? That it's not being applied (preferably by confirming that the text prompt sent to the LLM actually doesn't contain the voice mode prompt)?

**Reply by Classic298** [reply]
It does work for me. Try something outlandish like making it talk like donald trump and you'll see
